documentation/source/newsfragments/2059.feature.rst

```diff
@@ -0,0 +1 @@
+Added :class:`~cocotb.types.Logic` and :class:`~cocotb.types.Bit` modeling datatypes.
```

Comment:
```suggestion
Added modeling datatypes
:class:`~cocotb.types.Logic`, :class:`~cocotb.types.Bit`,
:class:`~cocotb.types.Array`, :class:`~cocotb.types.Range`,
:class:`~cocotb.types.LogicArray` and :class:`~cocotb.types.BitArray`.
```

Reply:
These are going to be split up into multiple PRs and newsfragments.